tools/gc_derivatization/gc_derivatization.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="gc_derivatization" name="Gas chromatography derivatization" version="@TOOL_VERSION@+galaxy1" profile="21.09">
+    <description>This tool perform MeOX and TMS derivatization</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">gc-meox-tms</requirement>
+    </requirements>
```

Comment:
creator is not imported afaics

Reply:
Imported [1515438](https://github.com/RECETOX/galaxytools/pull/503/commits/151543806594e8e6d01cce558b60a57bc6327ee3)